### ISSUE TITLE
Interoperability with PyVista

### ISSUE THREAD

**Issue by banesullivan** [sdorkenw/MeshParty#33]
Would it make sense to build interoperability between MeshParty and PyVista? Maybe for 3D viz at the least?

I see you already have a VTK dependency... PyVista is simply a wrapper/helper library for VTK mesh/grid types that has full support for 3D rendering, analysis, and filtering.

**Comment by fcollman** [comment]
yes I think that I would like to basically remove much of what is in trimesh_vtk and replace it with pyvista and/or vtk plotter style commands.  Certainly maintaining multiple implementation of numpy>vtk representations doesn't make sense.
